_data/osdf_namespace_metadata/ospool-uc-shared-project.yaml

```
@@ -25,4 +23,6 @@ namespace:
 thirtyDayReads: 21723475821543
 oneYearReads: 6775986102458060
 organizationUrl: https://www.uchicago.edu/en
+repositoryUrl: 
+  url: https://osg-htc.org/collaboration-support/
```

```suggestion
repositoryUrl: null
```

I would keep this null, this page doesn't have anything about datasets that this namespace might contain.